### PR TITLE
Add documentation of the inferno new command

### PR DESCRIPTION
**Pull request by mrnosal**
Added docs for the `inferno new` command to /docs/getting-started and /docs/getting-started/inferno-cli

# Summary
Now Getting Started mentions using `inferno new` to create Test Kit template as option to cloning. 
Inferno CLI page lists inferno new in the command table and includes paragraphs describing the feature.

# Testing Guidance
Verify links to the Inferno CLI page on the Getting Started page are correct. 
Verify inferno new is in CLI command table
Verify description of using inferno new is correct on CLI page.

### CODE REVIEW

File: docs/getting-started/inferno-cli.md [hunk 1]
```diff
@@ -25,10 +24,37 @@ The commands available include:
...
+It will automatically install the necessary Gem files used by Inferno Core and run the
+database migrations to bring the database up to date. Run `inferno new --help` for more
+options and examples.
+
+Note that this command follows the naming convention used by Ruby gems. The command
+`inferno new MyTestKit` will create a new folder named `my-test-kit`. Inside the 
+`my-test-kit/lib` folder, it will create the subdirectory `my_test_kit` for your test files, 
+and the main Ruby file named `my_test_kit.rb`. The main module in the `.rb` file will be named `MyTestKit`.
+
+In `my-test-kit/spec`, it will create the subdirectory `my_test_kit` and in the main `patient_group_spec.rb` RSpec file, it sets the module name to `MyTestKit::PatientGroup`.
```
Comment (Shaumik-Ashraf) [inline docs/getting-started/inferno-cli.md:56]: I would reword this as:
> The new test kit supports [RSpec](https://rspec.info) testing on the test kit itself. It also includes a working example in `my-test-kit/spec/patient_group_spec.rb`

Honestly using RSpec ontop of Inferno is very complicated so I wouldn't expect new users to try this. Using RSpec is more of a "if you know you know" sort of thing.

File: docs/getting-started/inferno-cli.md [hunk 2]
```diff
@@ -25,10 +24,37 @@ The commands available include:
...
+
+It will create a `my_test_kit.gemspec` file, and set the names of any authors supplied using the `-a` option, and set the name, summary and description values to your Test Kit name.
```
Comment (Shaumik-Ashraf) [inline docs/getting-started/inferno-cli.md:58]: I would add:
> You should manually inspect this file if you want to [publish or distribute the test kit](https://inferno-framework.github.io/docs/distributing-tests.html).

File: docs/getting-started/index.md [hunk 3]
```diff
@@ -69,9 +71,20 @@ WSL may be having networking issues.  To resolve this, you can follow the steps
...
+method ensures you will always be starting with the latest Test Kit Template. You can
+either clone this repository directly, or click the green "Use this template" button in GitHub
+to create your own repository based on the Inferno Template.
+
+The second method is to use the Inferno Command Line Interface. After installing
+the `inferno_core` gem, run `inferno new TestKitName` to create a new Test Kit
+Template in a directory named `test-kit-name`. See how to [create a new Test
+Kit](/docs/getting-started/inferno-cli.html#creating-a-new-test-kit) using the Inferno CLI for more
+information and options.
+
```
Comment (Shaumik-Ashraf) [inline docs/getting-started/index.md:85]: I'd like copy-pasteable code blocks:
```
gem install inferno_core && inferno new my-test-kit
```

File: docs/getting-started/index.md [hunk 4]
```diff
@@ -69,9 +71,20 @@ WSL may be having networking issues.  To resolve this, you can follow the steps
...
+1. Create your Test Kit by cloning the [Inferno Template](https://github.com/inferno-framework/inferno-template)
+   or by using the `inferno new` command.
 1. Run `bundle install` to install dependencies.
 1. Run `bundle exec inferno migrate` to set up the database.
```
Comment (Shaumik-Ashraf) [inline docs/getting-started/index.md:89]: These could also be code blocks instead of inline-code for easy copy-paste, but if they're any plans for a quickstart guide that would be the place to put these instead.

File: docs/getting-started/inferno-cli.md [hunk 5]
```diff
@@ -25,10 +24,37 @@ The commands available include:
...
 | `inferno services stop` | Stops the background services for Inferno |
 | `inferno start` | Starts Inferno |
-| `inferno suite SUBCOMMAND ...ARGS` | Performs suite-based operations. The available subcommands are `describe`, `help`, and `input_template`.|
+| `inferno suite SUBCOMMAND [...ARGS]` | Performs suite-based operations. The available subcommands are `describe`, `help`, and `input_template`.|
 | `inferno suite help SUBCOMMAND` | View details on the suite subcommands. |
 | `inferno suites` | Lists available test suites |
 {: .grid.command-table}
+
+## Creating a new Test Kit
+
```
Comment (Shaumik-Ashraf) [inline docs/getting-started/inferno-cli.md:37]: I would add a copy-paste code block somewhere here too:
```
inferno new MyTestKit --author "My Name" --author "My Friend" --implementation-guide hl7.fhir.us.core@6.1.0
```

File: docs/getting-started/inferno-cli.md [hunk 6]
```diff
@@ -25,10 +24,37 @@ The commands available include:
...
+[Inferno Template](https://github.com/inferno-framework/inferno-template) repository, as it automatically sets the names of files and folders to match the Test Kit name.  
+It will automatically install the necessary Gem files used by Inferno Core and run the
+database migrations to bring the database up to date. Run `inferno new --help` for more
+options and examples.
+
+Note that this command follows the naming convention used by Ruby gems. The command
+`inferno new MyTestKit` will create a new folder named `my-test-kit`. Inside the 
+`my-test-kit/lib` folder, it will create the subdirectory `my_test_kit` for your test files, 
+and the main Ruby file named `my_test_kit.rb`. The main module in the `.rb` file will be named `MyTestKit`.
+
```
Comment (Shaumik-Ashraf) [inline docs/getting-started/inferno-cli.md:55]: I would add a part that mentions how to specify implementation guides, i.e:
> The `--implementation-guide` or `-i` option will look at an absolute file path if its prefixed by `file://` or lookup the name in the [official FHIR package registry](https://packages.fhir.org)

**Review by Shaumik-Ashraf — COMMENTED** [review]
Glad to see this thing is finally reaching the official website! I added some comments but otherwise looks good.

File: docs/getting-started/index.md (hunk 3 above)
Comment (mrnosal) [reply docs/getting-started/index.md:85]: I like copy buttons on code blocks too, but I think I'll open a separate ticket for that functionality.

File: docs/getting-started/inferno-cli.md (hunk 5 above)
Comment (mrnosal) [reply docs/getting-started/inferno-cli.md:37]: added the code block, but will add copy-paste button separately

**Review by Shaumik-Ashraf — APPROVED** [review]
Looks good, thanks @mrnosal!
